packages/flutter/lib/src/material/banner.dart

```diff
@@ -25,21 +61,24 @@ import 'theme.dart';
 ///     appBar: AppBar(
 ///       title: const Text('The MaterialBanner is below'),
 ///     ),
-///     body: const MaterialBanner(
-///       padding: EdgeInsets.all(20),
-///       content: Text('Hello, I am a Material Banner'),
-///       leading: Icon(Icons.agriculture_outlined),
-///       backgroundColor: Color(0xFFE0E0E0),
-///       actions: <Widget>[
-///         TextButton(
-///           child: Text('OPEN'),
-///           onPressed: null,
-///         ),
-///         TextButton(
-///           child: Text('DISMISS'),
-///           onPressed: null,
+///     body: Center(
```

Comment:
Since we are maintaining backwards compatibility here and allowing for static banners, maybe we can leave this sample and add a new one that uses the new functionality.